src/cli.yaml

```diff
@@ -82,6 +82,28 @@ subcommands:
             about: Sets the output file to use
             required: true
             index: 2
+  - update:
+      about: >
+          Attempts to update an exsiting Valheim server installation. By
```

Comment:
existing

Reply:
Thanks for looking over things! Fixed in e0d5351b72fdb57865783850e2fc6afa27a433b1